dangerzone/gui/main_window.py

```diff
@@ -60,6 +61,11 @@
 HAMBURGER_MENU_SIZE = 30
 
 
+WARNING_MESSAGE = """<p><b>Warning:</b> You are not receiving security updates because systems with Python 3.8 are no longer supported. 
+Please upgrade to a more recent version of your operating system to use an updated version of Dangerzone. While Dangerzone aims to protect 
+you from malicious documents, using an unsupported version may expose you to known security vulnerabilities.</p>"""
```

Comment:
I'd maybe write something like this:

```suggestion
WARNING_MESSAGE = """<p><b>Warning:</b> Ubuntu Focal systems and their derivatives will stop being supported in subsequent Dangerzone releases. We encourage you to upgrade to a more recent version of your operating system in order to get security updates.</p>"""
```

There are two things I wanted to point out here:
1. It's not just a Python 3.8 issue. For instance, running Dangerzone from source on Ubuntu Focal uses Python 3.9, but we'll ditch this version as a whole.
2. Some people may use Dangerzone in Linux Mint systems (we've received bug reports)

Reply:
Perfect! Done in ce7c6defea8d1bcb7b012eadfa338999c123f9fe

![image](https://github.com/user-attachments/assets/f143e49f-e391-4bf2-8afd-e0faa3ad21db)